packages/app/src/domain/variants/variants-table-tile/index.tsx

```diff
@@ -51,35 +51,82 @@ export function VariantsTableTile({
     date_end_unix: number;
     date_of_insertion_unix: number;
   } | null;
-  children?: ReactNode | null;
+  children?: ReactNode;
+}) {
+  if (!isPresent(data) || !isPresent(dates)) {
+    return (
+      <FullscreenChartTile>
+        <Heading level={3}>{text.titel}</Heading>
+        <Box maxWidth="maxWidthText">
```

Comment:
This looks like a mistake did you mean to write this?
```suggestion
        <Box maxWidth={maxWidthText}>
```

Reply:
Nope, this is theme property, not an actual value. (This is being used in a lot of other places as well, by the way, I didn't make this up here)